lib/jsdom/living/svg/List.js

```diff
@@ -0,0 +1,164 @@
+"use strict";
+
+const DOMException = require("domexception");
+const idlUtils = require("../generated/utils");
+const { attach, detach } = require("./basic-types");
+
+// https://svgwg.org/svg2-draft/types.html#ListInterfaces
+
+// Child classes must implement:
+// - _getValue(), which returns an array of values.
+// - get _needsResync()
+// - _reserialize()
+class List {
```

Comment:
Maybe call it SVGListBase or something? Even though it's namespaced by module name/folder name, it seems a bit nicer to me that way.